deploy/longhorn-images.txt

```diff
@@ -1,7 +1,7 @@
 longhornio/csi-attacher:v3.4.0
 longhornio/csi-provisioner:v2.1.2
-longhornio/csi-resizer:v1.2.0
-longhornio/csi-snapshotter:v3.0.3
+longhornio/csi-resizer:v1.3.0
+longhornio/csi-snapshotter:v5.0.1
```

Comment:
Just curious. Why not upgrade to latest versions?

Reply:
v5.0.1 deprecated but is still serving CSI snapshot `v1beta1` while v.6.0.1 stopped serving it. This gives users some buffer to migrate their CSI snapshot manifest/script that is using `v1beta1`. What do you think?

Reply:
> This gives users some buffer to migrate their CSI snapshot manifest/script that is using v1beta1. 

Got it. Agreed the buffer. Thank you.

Reply:
@innobead for the release note we also need to make sure that users have cleaned up their potentially previously invalid csi snapshots, since v4.1 they will no longer be served and I think there might be problems with the controller in case of invalid objects.

REF: https://github.com/kubernetes-csi/external-snapshotter#snapshot-v1-apis

Reply:
:point_up: let's make this info in the source ticket description as well to avoid any missing.

Reply:
@PhanLe1010 Why didn't we upgrade `external-provisioner`? I saw the latest 3.3 can have min 1.20 supported. The same question for `[external-attacher](https://kubernetes-csi.github.io/docs/external-attacher.html#csi-external-attacher)`

cc @weizhe0422

Reply:
I followed this steps to update the components, In steps 5, it talked about how to select csi sidecars components. 

Longhorn v1.4.0 supports min K8s v1.21. On the other hand, k8s v.1.21 is speaking CSI spec v1.3.0 version.  Therefore, Longhorn components also need to support min csi spec version 1.3.0 so that we can talk to K8s v.1.21. 

Using the min csi spec version v1.3.0, we cannot upgrade external-provisioner to newer than v2.1.2. Same thing for other csi components